src/test/java/apicurito/tests/configuration/templates/ApicuritoTemplate.java

```diff
@@ -62,38 +66,110 @@ public static void deployUsingTemplate() {
         // process & create
         KubernetesList processedTemplate = OpenShiftUtils.getInstance().recreateAndProcessTemplate(template, templateParams);
         for (HasMetadata hasMetadata : processedTemplate.getItems()) {
-            if (!(hasMetadata instanceof CronJob)) {
-                OpenShiftUtils.getInstance().createResources(hasMetadata);
-            }
+            OpenShiftUtils.getInstance().createResources(hasMetadata);
         }
     }
 
+    private static void deployUsingGoOperator() {
+        log.info("Deploying using GO operator");
+        OpenShiftUtils.getInstance().clean();
+        OpenShiftUtils.getInstance().waiters().isProjectClean().waitFor();
+
+        deployCrd();
+        deployService();
+        deployRole();
+        deployRoleBinding();
+        deployOperator();
+        deployCr();
+    }
+
+    private static void deployCrd() {
+        log.info("Deploying CRD from " + TestConfiguration.apicuritoOperatorCrdUrl());
+        final String output = OpenShiftUtils.binary().execute(
+                "create",
+                "-n", TestConfiguration.openShiftNamespace(),
+                "-f", TestConfiguration.apicuritoOperatorCrdUrl()
+        );
+    }
+
+    private static void deployCr() {
+        log.info("Deploying CR from " + TestConfiguration.apicuritoOperatorCrUrl());
+        final String output = OpenShiftUtils.binary().execute(
+                "apply",
+                "-n", TestConfiguration.openShiftNamespace(),
+                "-f", TestConfiguration.apicuritoOperatorCrUrl()
+        );
+    }
+
+    private static void deployService() {
+        log.info("Deploying Service from " + TestConfiguration.apicuritoOperatorServiceUrl());
+        final String output = OpenShiftUtils.binary().execute(
+                "create",
+                "-n", TestConfiguration.openShiftNamespace(),
+                "-f", TestConfiguration.apicuritoOperatorServiceUrl()
+        );
+    }
+
+    private static void deployRole() {
+        log.info("Deploying Role from " + TestConfiguration.apicuritoOperatorRoleUrl());
+        final String output = OpenShiftUtils.binary().execute(
+                "create",
+                "-n", TestConfiguration.openShiftNamespace(),
+                "-f", TestConfiguration.apicuritoOperatorRoleUrl()
+        );
+    }
+
+    private static void deployRoleBinding() {
+        log.info("Deploying Role binding from " + TestConfiguration.apicuritoOperatorRoleBindingUrl());
+        final String output = OpenShiftUtils.binary().execute(
+                "create",
+                "-n", TestConfiguration.openShiftNamespace(),
+                "-f", TestConfiguration.apicuritoOperatorRoleBindingUrl()
+        );
+    }
+
+    private static void deployOperator() {
```

Comment:
These 6 methods seem to be quite repetitive, could these be squashed?

Reply:
5 squashed, CR will be reworked in next PR

Reply:
looks better, thanks